cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/connector/resource/AzureDatabaseResourceService.java

```diff
@@ -107,7 +107,11 @@ public List<CloudResourceStatus> buildDatabaseResourcesForLaunch(AuthenticatedCo
         createTemplateResource(persistenceNotifier, cloudContext, stackName);
         Deployment deployment;
         try {
-            deployDatabaseServer(stackName, resourceGroupName, template, client);
+            if (client.templateDeploymentExists(resourceGroupName, stackName)) {
+                LOGGER.debug("The database server {} is already exist.", stack);
+            } else {
+                deployDatabaseServer(stackName, resourceGroupName, template, client);
+            }
```

Comment:
you might want to consider moving this check into the deploy method, or add it to `upgradeDatabaseServer` too, as I suspect it can happen during upgrade too